README.md

```diff
@@ -38,8 +38,8 @@ Kubefile:
CAgICAgICAgICBydW5Bc1VzZXI6IDEwMDEKICAgICAgICAgICAgcnVuQXNHcm91cDogMjAwMQogICAgICB2b2x1bWVzOgogICAgICAgIC0gbmFtZToga3ViZXJuZXRlcy1kYXNoYm9hcmQtY2VydHMKICAgICAgICAgIHNlY3JldDoKICAgICAgICAgICAgc2VjcmV0TmFtZToga3ViZXJuZXRlcy1kYXNoYm9hcmQtY2VydHMKICAgICAgICAtIG5hbWU6IHRtcC12b2x1bWUKICAgICAgICAgIGVtcHR5RGlyOiB7fQogICAgICBzZXJ2aWNlQWNjb3VudE5hbWU6IGt1YmVybmV0ZXMtZGFzaGJvYXJkCiAgICAgIG5vZGVTZWxlY3RvcjoKICAgICAgICAia3ViZXJuZXRlcy5pby9vcyI6IGxpbnV4CiAgICAgICMgQ29tbWVudCB0aGUgZm9sbG93aW5nIHRvbGVyYXRpb25zIGlmIERhc2hib2FyZCBtdXN0IG5vdCBiZSBkZXBsb3llZCBvbiBtYXN0ZXIKICAgICAgdG9sZXJhdGlvbnM6CiAgICAgICAgLSBrZXk6IG5vZGUtcm9sZS5rdWJlcm5ldGVzLmlvL21hc3RlcgogICAgICAgICAgZWZmZWN0OiBOb1NjaGVkdWxlCgotLS0KCmtpbmQ6IFNlcnZpY2UKYXBpVmVyc2lvbjogdjEKbWV0YWRhdGE6CiAgbGFiZWxzOgogICAgazhzLWFwcDogZGFzaGJvYXJkLW1ldHJpY3Mtc2NyYXBlcgogIG5hbWU6IGRhc2hib2FyZC1tZXRyaWNzLXNjcmFwZXIKICBuYW1lc3BhY2U6IGt1YmVybmV0ZXMtZGFzaGJvYXJkCnNwZWM6CiAgcG9ydHM6CiAgICAtIHBvcnQ6IDgwMDAKICAgICAgdGFyZ2V0UG9ydDogODAwMAogIHNlbGVjdG9yOgogICAgazhzLWFwcDogZGFzaGJvYXJkLW1ldHJpY3Mtc2NyYXBlcgoKLS0tCgpraW5kOiBEZXBsb3ltZW50CmFwaVZlcnNpb246IGFwcHMvdjEKbWV0YWRhdGE6CiAgbGFiZWxzOgogICAgazhzLWFwcDogZGFzaGJvYXJkLW1ldHJpY3Mtc2NyYXBlcgogIG5hbWU6IGRhc2hib2FyZC1tZXRyaWNzLXNjcmFwZXIKICBuYW1lc3BhY2U6IGt1YmVybmV0ZXMtZGFzaGJvYXJkCnNwZWM6CiAgcmVwbGljYXM6IDEKICByZXZpc2lvbkhpc3RvcnlMaW1pdDogMTAKICBzZWxlY3RvcjoKICAgIG1hdGNoTGFiZWxzOgogICAgICBrOHMtYXBwOiBkYXNoYm9hcmQtbWV0cmljcy1zY3JhcGVyCiAgdGVtcGxhdGU6CiAgICBtZXRhZGF0YToKICAgICAgbGFiZWxzOgogICAgICAgIGs4cy1hcHA6IGRhc2hib2FyZC1tZXRyaWNzLXNjcmFwZXIKICAgICAgYW5ub3RhdGlvbnM6CiAgICAgICAgc2VjY29tcC5zZWN1cml0eS5hbHBoYS5rdWJlcm5ldGVzLmlvL3BvZDogJ3J1bnRpbWUvZGVmYXVsdCcKICAgIHNwZWM6CiAgICAgIGNvbnRhaW5lcnM6CiAgICAgICAgLSBuYW1lOiBkYXNoYm9hcmQtbWV0cmljcy1zY3JhcGVyCiAgICAgICAgICBpbWFnZToga3ViZXJuZXRlc3VpL21ldHJpY3Mtc2NyYXBlcjp2MS4wLjYKICAgICAgICAgIHBvcnRzOgogICAgICAgICAgICAtIGNvbnRhaW5lclBvcnQ6IDgwMDAKICAgICAgICAgICAgICBwcm90b2NvbDogVENQCiAgICAgICAgICBsaXZlbmVzc1Byb2JlOgogICAgICAgICAgICBodHRwR2V0OgogICAgICAgICAgICAgIHNjaGVtZTogSFRUUAogICAgICAgICAgICAgIHBhdGg6IC8KICAgICAgICAgICAgICBwb3J0OiA4MDAwCiAgICAgICAgICAgIGluaXRpYWxEZWxheVNlY29uZHM6IDMwCiAgICAgICAgICAgIHRpbWVvdXRTZWNvbmRzOiAzMAogICAgICAgICAgdm9sdW1lTW91bnRzOgogICAgICAgICAgLSBtb3VudFBhdGg6IC90bXAKICAgICAgICAgICAgbmFtZTogdG1wLXZvbHVtZQogICAgICAgICAgc2VjdXJpdHlDb250ZXh0OgogICAgICAgICAgICBhbGxvd1ByaXZpbGVnZUVzY2FsYXRpb246IGZhbHNlCiAgICAgICAgICAgIHJlYWRPbmx5Um9vdEZpbGVzeXN0ZW06IHRydWUKICAgICAgICAgICAgcnVuQXNVc2VyOiAxMDAxCiAgICAgICAgICAgIHJ1bkFzR3JvdXA6IDIwMDEKICAgICAgc2VydmljZUFjY291bnROYW1lOiBrdWJlcm5ldGVzLWRhc2hib2FyZAogICAgICBub2RlU2VsZWN0b3I6CiAgICAgICAgImt1YmVybmV0ZXMuaW8vb3MiOiBsaW51eAogICAgICAjIENvbW1lbnQgdGhlIGZvbGxvd2luZyB0b2xlcmF0aW9ucyBpZiBEYXNoYm9hcmQgbXVzdCBub3QgYmUgZGVwbG95ZWQgb24gbWFzdGVyCiAgICAgIHRvbGVyYXRpb25zOgogICAgICAgIC0ga2V5OiBub2RlLXJvbGUua3ViZXJuZXRlcy5pby9tYXN0ZXIKICAgICAgICAgIGVmZmVjdDogTm9TY2hlZHVsZQogICAgICB2b2x1bWVzOgogICAgICAgIC0gbmFtZTogdG1wLXZvbHVtZQogICAgICAgICAgZW1wdHlEaXI6IHt9Cg==" |base64  -d  > recommended.yaml
```

Comment:
OH, put those text in README is Terrible. I think you can add a dashboard application: [helm example](https://github.com/alibaba/sealer/tree/main/applications/helm/v3.6.0)

Reply:
OK, I try sealer follow readme found this 
 url "https://raw.githubusercontent.com/kubernetes/dashboard/v2.2.0/aio/deploy/recommended.yaml" sometimes timeout, so I think change offline type for better experience